src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -111,6 +122,37 @@ private async Task HandleSaveTableChangesRequest(SaveTableChangesRequestParams r
                        });
         }
 
+        private void HandleAddItemRequest(ProcessTableDesignerEditRequestParams requestParams)
+        {
+            var property = requestParams.TableChangeInfo.Property;
+            if (property.GetType() == typeof(string))
+            {
+                string propertyName = property as string;
```

Comment:
You can also just do 

``` csharp
string propertyName = property as string;
if(propertyName != null)
{
...
```

That's typically how I see these types of checks being done.